ts/package.json

```diff
@@ -15,25 +10,27 @@
     "type": "git",
     "url": "https://github.com/project-serum/anchor.git"
   },
-  "publishConfig": {
-    "access": "public"
-  },
-  "engines": {
-    "node": ">=11"
-  },
+  "license": "(MIT OR Apache-2.0)",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/browser/index.js",
+  "types": "dist/cjs/index.d.ts",
   "scripts": {
     "build": "rimraf dist/ && yarn build:node && yarn build:browser",
-    "build:node": "tsc && tsc -p tsconfig.cjs.json",
     "build:browser": "rollup --config",
-    "lint:fix": "prettier src/** tests/** -w",
-    "lint": "prettier src/** tests/** --check",
-    "watch": "tsc -p tsconfig.cjs.json --watch",
-    "prepublishOnly": "yarn build",
+    "build:node": "tsc && tsc -p tsconfig.cjs.json",
     "docs": "typedoc --excludePrivate --includeVersion --out ../docs/src/.vuepress/dist/ts/ --readme none src/index.ts",
-    "test": "jest tests --detectOpenHandles"
+    "lint": "prettier src/** tests/**/*.ts --check",
+    "lint:fix": "prettier src/** tests/**/*.ts -w",
+    "prepublishOnly": "yarn build",
+    "test": "jest tests --detectOpenHandles",
+    "test:live": "TEST_LIVE=1 yarn test",
+    "test:live-with-test-validator": "start-server-and-test 'solana-test-validator --reset --quiet' http://127.0.0.1:8899/health test:live",
+    "watch": "tsc -p tsconfig.cjs.json --watch"
   },
   "dependencies": {
     "@project-serum/borsh": "^0.2.5",
+    "@solana/buffer-layout": "^3.0.0",
```

Comment:
Can we use the existing buffer-layout dependency (below) instead of adding this?

Reply:
Yeah, we can rid it :+1: